config/manager/kustomization.yaml

```diff
@@ -1,8 +1,17 @@
 resources:
 - manager.yaml
+
```

Comment:
There was a TODO to change the Always back. I think it might be better to set the default to IfNotPresent and supply this "patch" as commented out.